myhw-week1/created-data.js

```diff
@@ -0,0 +1,181 @@
+const mysql = require('mysql');
+const connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'hyfuser',
+  password: 'hyfpassword',
+  database: 'userdb'
+});
+
+connection.connect();
+
+// creating database world
+let sql = "create database if not exists world"
+connection.query(sql, function (error, results) {
+  if (error) {
+    throw error;
+  }
+  console.log(results);
+});
+
+connection.query("use world")
+
+let countries_query = "create table if not exists countries(country_name varchar(20) unique, country_population bigint, continent varchar(20), surface_area int,primary key(country_name))"
```

Comment:
can country_name be a primary key ?

Reply:
yes why not it is uniquely identifies  and contains UNIQUE values, right?

Reply:
I will accept it for this homework.